File: Example/TableViewFactory/Cells/BankCell.m
//
//  BankCell.m
//  DTTableViewManager
//
//  Created by Denys Telezhkin on 08.12.13.
//  Copyright (c) 2013 Denys Telezhkin. All rights reserved.
//

#import "BankCell.h"
#import "Bank.h"

@implementation BankCell

-(void)updateWithModel:(id)model
{
    Bank * bank = model;
    self.textLabel.text = bank.name;
    self.detailTextLabel.text = bank.city;
}

@end

File: Example/Example controllers/Search/BankCell.swift
//
//  BankCell.swift
//  DTTableViewManager
//
//  Created by Denys Telezhkin on 20.08.15.
//  Copyright (c) 2015 Denys Telezhkin. All rights reserved.
//

import Foundation
import UIKit
import DTModelStorage

class BankCell : UITableViewCell, ModelTransfer
{
    func updateWithModel(_ model: Bank) {
        self.textLabel?.text = model.name
        self.detailTextLabel?.text = model.city
    }
}
